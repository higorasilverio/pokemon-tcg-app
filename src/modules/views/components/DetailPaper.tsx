import styles from '@/styles/DetailPaper.module.scss'
import { ReactNode } from 'react'

const DetailPaper = ({ children }: { children: ReactNode }) => {
  return <div className={styles.paper}>{children}</div>
}

export default DetailPaper
