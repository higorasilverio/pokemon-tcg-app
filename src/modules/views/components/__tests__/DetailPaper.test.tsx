import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import DetailPaper from '../DetailPaper'

describe('DetailPaper', () => {
  it('renders unchanged', () => {
    const { container } = render(<DetailPaper>Detail</DetailPaper>)

    expect(container).toMatchSnapshot()
  })

  it('should render correctly', () => {
    render(<DetailPaper>Detail</DetailPaper>)

    const text = screen.getByText(/detail/i)

    expect(text).toBeInTheDocument()
  })
})
