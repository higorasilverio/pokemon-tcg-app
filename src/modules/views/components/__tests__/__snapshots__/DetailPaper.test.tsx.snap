// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailPaper renders unchanged 1`] = `
<div>
  <div
    class="paper"
  >
    Detail
  </div>
</div>
`;
